CMakeLists.txt

```diff
@@ -545,6 +547,18 @@ ENDIF()
 # checks, like in configure.ac.
 ###
 
+##
+# Check to see if compiler supports
+# -fallow-argument-mismatch-flag introduced in gcc 10.
+#
+# See https://github.com/Unidata/netcdf-fortran/issues/212
+##
+check_Fortran_compiler_flag("-fallow-argument-mismatch" COMPILER_HAS_ALLOW_ARGUMENT_MISMATCH)
```

Comment:
For the NAG compiler, it is `-mismatch_all`, as written here: https://github.com/Unidata/netcdf-fortran/issues/218

Reply:
Excellent, I'll add a similar check!